Sources/ProjectDescription/Config.swift

```diff
@@ -23,8 +45,11 @@ public struct Config: Codable, Equatable {
         /// Tuist generates the project with the specific development region.
         case developmentRegion(String)
 
-        /// Tuist generates the project only with custom specified schemes, autogenerated default
-        case disableAutogeneratedSchemes
+        /// Options controlling how / if Tuist automatically generates schemes
```

Comment:
Should we enhance the doc to explain the default if nothing is specified? Both here and in the md file